kategory/src/main/kotlin/kategory/data/StateT.kt

```diff
@@ -22,6 +22,9 @@ class StateT<F, S, A>(
         inline operator fun <reified F, S, A> invoke(noinline run: StateTFun<F, S, A>, MF: Monad<F> = monad<F>()): StateT<F, S, A> =
                 StateT(MF, MF.pure(run))
 
+        fun <F, S, A> invokeF(runF: StateTFunKind<F, S, A>, MF: Monad<F>): StateT<F, S, A> =
```

Comment:
Sorry but can't. All the StateT class instance methods delegate their behavior on this one, so reifying the type here means we would need to make all of them inline reified also. I think that was the main reason for it to not be in the companion object, may be. But even tho, I think it's better placed inside of the comp obj.